### PR TITLE
docs(components): [time-picker] use new display tag

### PR DESCRIPTION
**Pull request by wzc520pyfm**
Related: #10754 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by btea — APPROVED** [review]

File: docs/en-US/component/time-picker.md [hunk 1]
```diff
@@ -49,24 +49,24 @@ time-picker/range
...
 | is-range              | whether to pick a time range                             | ^[boolean]                                                                                      | false       |
 | arrow-control         | whether to pick time using arrow buttons                 | ^[boolean]                                                                                      | false       |
-| popper-class          | custom class name for TimePicker's dropdown              | ^[string]                                                                                       | —           |
-| range-separator       | range separator                                          | ^[string]                                                                                       | '-'         |
+| popper-class          | custom class name for TimePicker's dropdown              | ^[string]                                                                                       | ''          |
+| range-separator       | range separator                                          | ^[string]                                                                                       | -           |
```
Comment (chenxch) [inline docs/en-US/component/time-picker.md:64]: `-` Connectors also need to keep quotes for easy distinction

File: docs/en-US/component/time-picker.md [hunk 2]
```diff
@@ -49,24 +49,24 @@ time-picker/range
...
-| name                  | same as `name` in native input                           | ^[string]                                                                                       | —           |
-| label ^(a11y)         | same as `aria-label` in native input                     | ^[string]                                                                                       | —           |
+| default-value         | optional, default date of the calendar                   | ^[Date] / ^[object]`[Date, Date]`                                                               | —           |
+| id                    | same as `id` in native input                             | ^[string] / ^[object]`[string, string]`                                                         | —           |
+| name                  | same as `name` in native input                           | ^[string]                                                                                       | ''          |
+| label ^(a11y)         | same as `aria-label` in native input                     | ^[string]                                                                                       | undefined   |
```
Comment (chenxch) [inline docs/en-US/component/time-picker.md:69]: `undefined` => `—`

File: docs/en-US/component/time-picker.md (hunk 1 above)
Comment (wzc520pyfm) [reply docs/en-US/component/time-picker.md]: updated

File: docs/en-US/component/time-picker.md (hunk 2 above)
Comment (wzc520pyfm) [reply docs/en-US/component/time-picker.md]: updated

**Review by tolking — APPROVED** [review]

**Pull request merged**